Node/src/TeamsMessage.ts

```diff
@@ -41,19 +41,75 @@ export enum MentionTextLocation {
   AppendText
 }
 
+/**
+ * @class
+ * At mention entity in message.
+ *
+ * @member {string} [type] at mention type, its value is always mention.
+ *
+ * @member {object} [mentioned] mentioned object with id, type and text value.
+ *
+ * @member {string} [text] text value to display in the message
+ *
+ */
+export class MentionEntity {
+  type: string;
+  mentioned: any;
+  text: string;
+}
+
+export class UserMention extends MentionEntity {
+    constructor(user: builder.IIdentity, text?: string) {
+      super();
+      this.type = 'mention';
+      if (text != null)
```

Comment:
This should be "if (text)", or else you admit empty strings.

Reply:
You should do the same checks as in addMentionToText. user has id, and either user has name or text is provided.

Reply:
fixed